MetricsDashboard.Contract/BooleanResultDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace MetricsDashboard.Contract
+{
+    public class BooleanResultDto
```

Comment:
We need to think about better name of this kind of tile, something lige StatusTile maybe?

Reply:
renamed to StatusDto